stdlib/OpenBLAS_jll/test/runtests.jl

```diff
@@ -1,8 +1,7 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-using Test, Libdl, OpenBLAS_jll
+using Test, Libdl, OpenBLAS_jll, LinearAlgebra
```

Comment:
Let's not use `LinearAlgebra` from `OpenBLAS_jll`; let's just inline `BLAS.@blasfunc` here.

Reply:
Do you mean like manually copy the relevant code?